example/hello_object_detection_ssd_node/box.js

```diff
@@ -0,0 +1,29 @@
+class ObjectDetectionPrediction {
+    constructor(x1, y1, x2, y2, probability, classId) {
```

Comment:
We follow 2 spaces indentation. Would you please follow [coding style](https://github.com/intel/inference-engine-node/blob/master/doc/coding_style.md) and run clang-format for it? Thanks!

Reply:
Fixed in commit: "Review fixes"